README.md

```diff
@@ -16,7 +16,7 @@ A React library to help you keep track of your modal state using the URL.
 
 [Documentation](https://react-url-modal.vercel.app)
 
-To create a new instance of modals you can import the `URLModal` and pass the modals you have in your application:
+To create a new instance of modals, import the `URLModal` and pass the modals you have in your application:
```

Comment:
I've removed _you can..._ in situations that explain a required flow/step to accomplish something.

IMO,  "you can" is meant to be used in scenarios where there are multiple options. eg "if you need to jump, you can press `X` or go to the trampoline")

Reply:
Makes perfect sense and good catch!